index.js

```diff
@@ -82,12 +84,18 @@ function processArgv(argv) {
     }
 
     if (module && typeof module.runCommand === 'function') {
+      const pkgName = pkgJson.name || dirName;
+
       if (verbose) {
-        const pkgName = pkgJson.name || dirName;
         logger.info(`Passing command to ${pkgName}`);
       }
 
-      module.runCommand(command, argv);
+      if (modulesCalled[pkgName]) {
+        logger.info(`Skipping multiple instances of ${pkgName}`);
```

Comment:
Should we also hide this message behind the `verbose` flag? Also curious if we shouldn't show the log "Passing command to..." if the module has been called already (to avoid confusion).

Reply:
I liked your feedback.  Incorporated it into latest commit.